src/Page/Article.elm

```diff
@@ -151,6 +151,7 @@ viewAddComment postingDisabled maybeUser =
                         [ class "form-control"
                         , placeholder "Write a comment..."
                         , attribute "rows" "3"
+                        , value commentText
```

Comment:
Interesting. Did not know that. 
I seems like there is no easy solution for that?